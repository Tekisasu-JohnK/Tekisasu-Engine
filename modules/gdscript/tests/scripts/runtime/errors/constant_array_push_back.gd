const array: Array = [0]

func test():
	array.push_back(0)
