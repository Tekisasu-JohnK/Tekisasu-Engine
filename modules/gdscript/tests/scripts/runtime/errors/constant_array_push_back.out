GDTEST_RUNTIME_ERROR
>> ERROR
>> on function: push_back()
>> core/variant/array.cpp
>> 253
>> Condition "_p->read_only" is true.
>> Array is in read-only state.
